### DISCUSSION TITLE
Any places to get datapacks for bees?

### DISCUSSION THREAD

**Discussion by grandogarndo** [Resourceful-Bees/ResourcefulBees#276] [General]
I have a fairly large custom modpack, and don't want to download a bunch of other ones and balance their bees to add them to mine. Is there a place to find a large amount of custom made bees? (like modded metals, etc.)

**Comment by Epic428** [comment]
Not really. There is our custom bee channel in discord. But generally people modify ones already used in bigger packs. We do however plan to enable some kind of custom bee sharing using github as the storage location after we release 1.0.0
